lib/bucky/test_equipment/selenium_handler/webdriver_handler.rb

```diff
@@ -56,7 +56,8 @@ def set_chrome_option(device_type)
             chrome_options['goog:chromeOptions']['mobileEmulation'] = mobile_emulation
           end
           chrome_options['goog:chromeOptions'][:args] << "--user-agent=#{@@config[:user_agent]}" if @@config[:user_agent]
-          chrome_options['goog:chromeOptions'][:args] << '--headless' if @@config[:headless]
+          chrome_options['goog:chromeOptions'][:args] << '--headless --disable-gpu' if @@config[:headless]
+          chrome_options['goog:chromeOptions'][:args].push(@@config[:chromedriver_flags]).flatten! unless @@config[:chromedriver_flags].nil?
```

Comment:
I have a question.
Why you use `push` and `flatten!`?
I think `concat` is better.
```suggestion
          chrome_options['goog:chromeOptions'][:args].concat(@@config[:chromedriver_flags]) unless @@config[:chromedriver_flags].nil?
```

Reply:
COOL